packages/web/public/icons/sprite.svg

```diff
@@ -360,5 +360,23 @@
+            fill="currentColor"
+          />
+        </g>
+      </g>
+    </symbol>
+    <symbol viewBox="0 0 14 14" fill="none" id="left-right">
+      <path
+        d="M12.8323 4.08559C12.8323 3.9612 12.7858 3.83631 12.691 3.74152L10.5365 1.74429C10.3003 1.52495 9.91675 1.69231 9.91675 2.01431L9.91675 3.49998L7.00008 3.49998C6.92278 3.49889 6.84604 3.51317 6.77431 3.542C6.70258 3.57082 6.63729 3.61361 6.58224 3.66789C6.52719 3.72217 6.48348 3.78684 6.45364 3.85816C6.4238 3.92947 6.40844 4.00601 6.40844 4.08332C6.40844 4.16062 6.4238 4.23716 6.45364 4.30847C6.48348 4.37979 6.52719 4.44447 6.58224 4.49874C6.63729 4.55302 6.70258 4.59581 6.77431 4.62464C6.84604 4.65346 6.92278 4.66774 7.00008 4.66665L9.91675 4.66665L9.91675 6.15574C9.91675 6.47833 10.3003 6.64568 10.5365 6.42576L12.691 4.42853C12.7858 4.33374 12.8323 4.20999 12.8323 4.08559ZM12.2501 9.91665C12.2501 9.59465 11.9887 9.33332 11.6667 9.33332C11.3447 9.33332 11.0834 9.59465 11.0834 9.91665C11.0834 10.2386 11.3447 10.5 11.6667 10.5C11.9887 10.5 12.2501 10.2386 12.2501 9.91665ZM9.91675 9.91665C9.91675 9.59465 9.65541 9.33332 9.33341 9.33332C9.01141 9.33332 8.75008 9.59465 8.75008 9.91665C8.75008 10.2386 9.01141 10.5 9.33341 10.5C9.65541 10.5 9.91675 10.2386 9.91675 9.91665ZM7.59139 9.90753C7.58897 9.75302 7.52536 9.60578 7.4145 9.49813C7.30364 9.39047 7.1546 9.3312 7.00008 9.33332L4.08341 9.33332L4.08341 7.84764C4.08341 7.52564 3.69987 7.35829 3.46362 7.57762L1.30916 9.57485C1.11958 9.76444 1.11958 10.0723 1.30916 10.2619L3.46362 12.2591C3.69987 12.479 4.08341 12.3117 4.08341 11.9891L4.08341 10.5L7.00008 10.5C7.07813 10.5011 7.15559 10.4864 7.22789 10.457C7.30019 10.4276 7.36585 10.384 7.42099 10.3288C7.47612 10.2735 7.51962 10.2078 7.54889 10.1354C7.57816 10.0631 7.59261 9.98558 7.59139 9.90753ZM5.25008 4.08332C5.25008 3.76132 4.98875 3.49998 4.66675 3.49998C4.34475 3.49998 4.08341 3.76132 4.08341 4.08332C4.08341 4.40532 4.34475 4.66665 4.66675 4.66665C4.98875 4.66665 5.25008 4.40532 5.25008 4.08332ZM2.91675 4.08332C2.91675 3.76132 2.65541 3.49998 2.33341 3.49998C2.01142 3.49998 1.75008 3.76132 1.75008 4.08332C1.75008 4.40532 2.01142 4.66665 2.33341 4.66665C2.65541 4.66665 2.91675 4.40532 2.91675 4.08332Z"
```

Comment:
Small reminder to delete any icons that are here but also in icons folder, to keep things clean